src/lib.js

```diff
@@ -305,7 +305,7 @@ module.exports.buckets = function(items, field, agg, aggregations) {
     return {
       key: key,
       doc_count: val,
-      selected: _.includes(key, agg.filters)
+      selected: helpers.includes_any_element(key, agg.filters)
```

Comment:
That's interesting. `_.includes` should work because key is scalar value like `x` and agg.filters is array like `['x','y']`.
Probably it should be `_.includes(agg.filters, key);` though (https://lodash.com/docs/4.17.15#includes)

What do you think ?

Reply:
Good catch! It worked in reverse before indeed with only 1 element, because `key` was the "collection" then. My bad!

Thanks! Fixed!